buildscripts/ci/linux/tools/make_appimage.sh

```diff
@@ -208,7 +208,8 @@ function fallback_library()
 # Report new additions at https://github.com/linuxdeploy/linuxdeploy/issues
 # or https://github.com/linuxdeploy/linuxdeploy-plugin-qt/issues for Qt libs.
 unwanted_files=(
-  # none
+  # https://github.com/musescore/MuseScore/issues/24068#issuecomment-2297823192
+  lib/libwayland-client.so.0
```

Comment:
It might also be worth mentioning `libwayland-client.so.0` (without `lib/` prefix)  in the `fallback_libraries` array further down. This way it will be loaded if it's not found on the system, which could prevent a crash.

You'll need to keep it here in `unwanted_files` (with `lib/` prefix) so that it doesn't also get packaged in the `lib` folder as well as the `fallback` folder.

Reply:
Making it `fallback` is potentially only worth doing on the 4.4 branch, whereas excluding it is worth doing on both branches.

Reply:
I'll add it to `fallback`. Why would that not necessary on master too?

Reply:
We don't know for sure that excluding it will cause a crash, so I want to find out one way or the other and then report the outcome to the Linuxdeploy developers.